packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from 'react-portal';
+import PopupContent from './_PopupContent';
+import PopupArrow from './_PopupArrow';
+import PopupOverlay from './_PopupOverlay';
+import TetherComponent from './_TetherComponent';
+import PopupUtils from './_PopupUtils';
+import './Popup.scss';
+
+const HEIGHT_KEYS = ['40', '80', '120', '160', '240', '320', '400', '480', '560', '640', '720', '800', '880'];
+const HEIGHT_VALUES = [40, 80, 120, 160, 240, 320, 400, 480, 560, 640, 720, 800, 880];
```

Comment:
However, I would probably just make height & width their own files as
```
const height = {
'40': 40, 
'80': 80,
}
export default height;
```

In this file, import height